dcs/task.py

```diff
@@ -316,23 +326,25 @@ class AttackGroup(Task):
     """
     Id = "AttackGroup"
 
-    def __init__(self, group_id=0, weapon_type: WeaponType = WeaponType.Auto):
+    def __init__(self, group_id=0, weapon_type: WeaponType = WeaponType.Auto,
+                 attack_limit: Optional[int] = None, group_attack=False,
+                 altitude_enabled=False, altitude=0, direction_enabled=False,
```

Comment:
Replace `altitude_enabled: bool = False` and `altitude: int = 0` with `altitude: Optional[int] = None`? Ditto for `direction`.

Reply:
you're right. i just copy pasted it without going into it deeper.
I adopted the parameters for AttackGroup. Left the other ones untouched as this would affect ppl using this Task alread with the parameters.

I also set the Default values for all Attack Tasks to 0 or 1 depending on the parameter instead of adding None to the actuall mission file inside the miz. DCS has not complained about it but its a wrong data type for it so i think they already have some error handling which takes care of it. But to be on the safe site and future proof i aded the correct default values.